CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Removed
 
+- `newBigNum` from module Ctl.Internal.Serialization as it is not used ([#1025](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1025))
+
```

Comment:
Generally we want to avoid including internal changes in the changelog, otherwise it'll get cluttered and it'll be of less use to users

```suggestion
```